beanie/odm/documents.py

```diff
@@ -1214,6 +1213,43 @@ def link_from_id(cls, id: Any):
         ref = DBRef(id=id, collection=cls.get_collection_name())
         return Link(ref, document_class=cls)
 
+    def bulk_writer(
```

Comment:
Could you please add a test for this method? In case something will be changed later it will prevent breaking changes